src/index.js

```diff
@@ -55,15 +68,15 @@ export default function urlJoin(...parts) {
     if (beforePathname) {
         url += beforePathname + (pathname ? '/' : '');
     // Otherwise start with the leading slash
-    } else if (options.leadingSlash) {
+    } else if (options.leadingSlash === true) {
```

Comment:
This change is unrelated with the PR and it’s not really necessary.

Reply:
Oh, I don't know why I did it, sorry! Just fixed it